### PR TITLE
[new][author] Add new library: cloudinary-core version 2.0.7 + autoupdate

### PR DESCRIPTION
**Pull request by cloudinary**
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

Refers to issue #7216

# Profile of the lib
 * Git repository (required): https://github.com/cloudinary/pkg-cloudinary-core
 * Mater Git repository (required): https://github.com/cloudinary/cloudinary_js
 * Official website (optional, not the repository): www.cloudinary.com
 * NPM package url (optional): cloudinary-core
 * GitHub / Bitbucket popularity (required): (package/master)
   - Count of stars: 3/143
   - Count of watchers: 7/22
   - Count of forks: 1/128
 * NPM download stats (optional):
   - Downloads in the last day: 39
   - Downloads in the last week: 135
   - Downloads in the last month: 780

# Essential checklist
 * [x] I'm the author of this library
   * [x] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [ ] Pull request is sending from a non-master branch with meaningful name.
 * [ ] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [ ] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in ~~commit message~~, comment.

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
@cloudinary would you please use `tools/fixFormat.js` to fix the indent of package.json?

File: ajax/libs/cloudinary-core/package.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    "responsive",
+    "jpg", "gif", "png", "tiff", "bmp", "pdf", "avi", "mpeg", "mpg", "webp", "webm"
+  ],
+  "filename": "cloudinary-core.min.js",
+  "npmName": "cloudinary-core",
+  "npmFileMap": [
+    {
+      "basePath": ".",
+      "files": [
+      "cloudinary-core.js",
```
Comment (PeterDaveHello) [inline ajax/libs/cloudinary-core/package.json]: What about using a single rule `"cloudinary-core*"` here?

File: ajax/libs/cloudinary-core/package.json [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    "multiple",
+    "picture",
+    "pictures",
+    "video",
+    "animation",
+    "thumbnails",
+    "CDN",
+    "crop", "scale", "color", "filter", "transparent", "background", "watermark",
+    "responsive",
+    "jpg", "gif", "png", "tiff", "bmp", "pdf", "avi", "mpeg", "mpg", "webp", "webm"
```
Comment (PeterDaveHello) [inline ajax/libs/cloudinary-core/package.json]: each keyword should be a new line.

File: ajax/libs/cloudinary-core/package.json (hunk 1 above)
Comment (PeterDaveHello) [inline ajax/libs/cloudinary-core/package.json]: Missing 2 spaces indent in the `files` array.

**Comment by cloudinary** [comment]
@PeterDaveHello Thanks for the comments. I'll update the PR. 
Do you know why CircleCI failed?

**Comment by PeterDaveHello** [comment]
@cloudinary Failed because oif the wrong indent :)

**Comment by PeterDaveHello** [comment]
@cloudinary btw, in this case, please do all the things in a single commit, thanks a lot.

**Comment by cloudinary** [comment]
> please do all the things in a single commit

Single _new_ commit or do you prefer me to `commit --amend` and `push -f`?

**Comment by PeterDaveHello** [comment]
@cloudinary Both fine, thanks.

**Comment by cloudinary** [comment]
@PeterDaveHello is there anything else we need to do to prepare this library?

**Comment by PeterDaveHello** [comment]
@cloudinary please rebase/fixup them into one commit, I might understand wrong about the one commit as I thought you will use that one commit to overwrite the old one. Thanks.

**Comment by cloudinary** [comment]
@PeterDaveHello Done. Thanks!

**Comment by PeterDaveHello** [comment]
Thanks @cloudinary

**Pull request merged**